cfme/tests/cli/test_appliance_update.py

```diff
@@ -115,13 +116,21 @@ def do_yum_update(appliance):
     with appliance.ssh_client as ssh:
         result = ssh.run_command('yum update -y', timeout=3600)
         assert result.success, f"update failed {result.output}"
+
+    output = str(result)
+    rpmnew_regex = r'warning: (.*) created as (.*\.rpmnew)'
+    groups = re.findall(rpmnew_regex, output)
+    map(lambda rpmold, rpmnew: ssh.run_command(f'mv {rpmnew} {rpmold}'), groups)
+    output = '\n'.join(filter(lambda x: not re.match(rpmnew_regex, x), result.output.splitlines()))
+
     appliance.evmserverd.start()
     appliance.wait_for_web_ui()
+    return output
 
 
-@pytest.mark.meta(automates=[1714236])
+@pytest.mark.meta(automates=[1714236], coverage=[1674055])
```

Comment:
The coverage keyword is for manual tests only. Add this BZ to the automates list.